src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/OidcTokenHandlerFactory.php

```diff
@@ -69,18 +76,13 @@ public function addConfiguration(NodeBuilder $node): void
                         ->info('Audience set in the token, for validation purpose.')
                         ->defaultNull()
                     ->end()
-                    ->arrayNode('signature')
+                    ->scalarNode('algorithm')
+                        ->info('Algorithm used to sign the token.')
+                        ->isRequired()
+                    ->end()
+                    ->scalarNode('key')
+                        ->info('JSON-encoded JWK used to sign the token (must contain a "kty" key).')
                         ->isRequired()
-                        ->children()
```

Comment:
removing one unneeded nesting level in the config /cc @vincentchalamon for the doc

Reply:
The `signature` level was useful in anticipation of https://github.com/symfony/symfony/issues/50441. But removing it is more MVP. Let's talk about it later on https://github.com/symfony/symfony/issues/50441